Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -40,42 +40,44 @@ class RawPCCProducer : public edm::global::EDProducer<edm::EndLuminosityBlockPro
   void globalEndLuminosityBlockProduce(edm::LuminosityBlock& lumiSeg, const edm::EventSetup& iSetup) const final;
   void produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const final;
 
-  edm::EDGetTokenT<reco::PixelClusterCounts> pccToken_;
-  const edm::ESGetToken<LumiCorrections, LumiCorrectionsRcd> lumiCorrectionsToken_;
-  const edm::EDPutTokenT<LumiInfo> putToken_;
+  edm::EDGetTokenT<reco::PixelClusterCounts> pccToken_;                              //input object labels
+  const edm::ESGetToken<LumiCorrections, LumiCorrectionsRcd> lumiCorrectionsToken_;  //background corrections from DB
+  const std::vector<int> modVeto_;      //The list of modules to skip in the lumi calc.
+  const bool applyCorr_;                //background corrections
   const std::string takeAverageValue_;  //Output average values
 
-  const std::vector<int> modVeto_;  //The list of modules to skip in the lumi calc.
+  const edm::EDPutTokenT<LumiInfo> putToken_;  //output object labels
 
+  const bool saveCSVFile_;  //produce csv lumi file
```

Comment:
Please put the comments above the code, it makes the code easier to read (I though code-format would do that, but seems like now)